### ISSUE TITLE
Unable to control card width

### ISSUE THREAD

**Issue by JohnnySSH** [thomasloven/lovelace-layout-card#154]
Hi,

I have been using the custom layout card for a few years now and it was working perfectly until the card was upgraded a while ago.

Something I noticed was that the width no longer covers 100% of the screen. From the Readme, it says that card defaults to 500px width so I'm wondering if I have hit this??

The code I am using is as follows:

```
views:
  - badges: []
    cards:
      - cards:
          - cards:
              - cards:
                [...]
                type: horizontal-stack
            type: vertical-stack
        column_width: 50%
        layout-type: custom:grid-layout
        max_columns: 2
        max_width: 100%
        min_columns: 1
        min_height: 3
        min_width: 30%
        view_layout: custom:grid-layout
        type: custom:layout-card
    panel: true
    path: menu
    title: MENU
```

I have attached a screenshot of what I'm seeing
![Screenshot at 2021-07-17 16-47-03](https://user-images.githubusercontent.com/6391309/126042454-5b61a632-3c1f-4171-b213-7db8a6006485.png)

This used to fit nicely across the screen with the older version of the card and would adapt to different resolutions. 

As a test I have also tried to set the width in the GUI editor as per the Readme's first example, unfortunately this did not work either. I tested using both "horizontal" and "grid" layouts.

How do I get the card to span the remaining area of the screen?

Thanks.

**Comment by thomasloven** [comment]
When updating a card, please make sure to always read the release notes.

Or at least the release titles, which in this case reads (in order):

- 2.0.0: **BREAKING!** Layout-card 2.0
- 2.1.0: **2.0 is BREAKING** - Make grid layouts fully responsive
- 2.1.1: **2.0 is BREAKING** - Improved grid layout responsiveness
- 2.2.0: **BREAKING!** layout is renamed view_layout
- 2.2.1: 2.2.0 IS **BREAKING**
- 2.2.2: **IF YOU'RE UPDATING FROM EARLIER THAN 2.0 - YOUR. CONFIG. WILL. NOT. WORK. WITHOUT. CHANGES.**
- 2.2.3: **PLEASE READ THE RELEASE NOTES FOR THE LAST COUPLE OF RELEASES**
- 2.3.0: Add gap-card and various tweaks
- 2.3.1: Fix for Home Assistant 2021.8

**Comment by thomasloven** [comment]
Also duplicate of **pinned** issue #111
